File: pinyougou-parent/pinyougou-interface/src/main/java/cn/itcast/core/service/SeckillService.java
package cn.itcast.core.service;

import cn.itcast.core.pojo.item.Item;
import cn.itcast.core.pojo.seckill.SeckillGoods;
import vo.SeckillGoodsVO;

import java.util.List;

public interface SeckillService {
    //回显当前商品
    SeckillGoodsVO findOne(Long goodsId);
    //根据库存id查询当前价格,库存数
    Item findByitemId(Long itemId);
    //申请秒杀
    void save(SeckillGoodsVO seckillGoodsVO);
    //回显秒杀数据
    List<SeckillGoods> findSeckillGoodsList(String name);
    //回显全部秒杀数据
    List<SeckillGoods> findAllSeckillGoodsList();
    //开始审核
    void updateStatus(Long[] ids, String status);
}

File: pinyougou-parent/pinyougou-web-manager/src/main/webapp/js/service/seckillService.js
//服务层
app.service('seckillService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findGoodsByGoodsId=function (id) {
        return $http.post('../seckill/findOne.do?goodsId='+id);
    }

    //联动查询价格,库存
	this.findByitemId=function (id) {
        return $http.post('../seckill/findByitemId.do?itemId='+id);
    }

    //保存秒杀
	this.save=function (seckillVO, item) {
		return $http.post('../seckill/save.do?',seckillVO);
    }

    //回显秒杀商品
    this.findSeckillGoodsList=function () {
        return $http.get('../seckill/findSeckillGoodsList.do');
    }

    //回显全部秒杀商品
    this.findAllSeckillGoodsList=function () {
        return $http.get('../seckill/findAllSeckillGoodsList.do');
    }

    //开始审核  审核通过 或是驳回
    this.updateStatus = function(ids,status){
        return $http.get('../seckill/updateStatus.do?ids='+ids+"&status="+status);
    }

});
